### PR TITLE
php72-xdebug (new formula, head only)

### PR DESCRIPTION
**Pull request by alcaeus**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [ ] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

According to @derickr [on twitter](https://twitter.com/derickr/status/902810592557375488), it will be a while until a stable release. Since xdebug now compiles on 7.2 I thought it would be good to add it as a head-only formula to allow people to test it. What do you think @javian?

### CODE REVIEW

**Comment by derickr** [comment]
I approve. Better to get people testing it now so I know of (more) issues before release too.

**Comment by alcaeus** [comment]
@javian any chance we could get this and my other pull requests merged soon?

**Comment by SMillerDev** [comment]
I'm fine with this, as a matter of fact I had it running like this locally already. @javian doesn't like HEAD formulas though so I'll let him decide

**Comment by derickr** [comment]
Would it help if I made 2.6.0alpha1 ? It's not quite ready for a finish, but it works mostly.

**Comment by alcaeus** [comment]
@javian any update on this? Would you be willing to make an exception in this case for testing purposes?

**Comment by marcospassos** [comment]
Guys, any update on this?

**Comment by alcaeus** [comment]
@javian any update?

File: Formula/php72-xdebug.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+require File.expand_path("../../Abstract/abstract-php-extension", __FILE__)
+
+class Php72Xdebug < AbstractPhp72Extension
+  init
+  desc "Provides debugging and profiling capabilities."
+  homepage "https://xdebug.org"
+  head "https://github.com/xdebug/xdebug.git"
+
+  bottle do
```
Comment (javian) [inline Formula/php72-xdebug.rb]: Please take out the bottle section

**Comment by derickr** [comment]
There will be a 7.2 compat release of Xdebug very soon — not a final one, but at least a beta.

**Comment by alcaeus** [comment]
@javian done, thanks! Didn't think about not wanting bottles for head-only formulae, makes sense!

**Comment by simPod** [comment]
PHP 7.2 is out http://php.net/archive/2017.php#id2017-11-30-1

**Comment by lcobucci** [comment]
Xdebug 2.6.0-alpha1 is out too =)

https://xdebug.org/updates.php#x_2_6_0alpha1

**Comment by alcaeus** [comment]
@lcobucci thanks for the hint. I've updated the formula to install 2.6.0alpha1.

@javian I've left out the bottle block - do you want me to re-add it for an alpha release or should people compile it themselves?

**Comment by javian** [comment]
@alcaeus the bottle block will be added by the bot automatically. If a new formula is created it should be left out and the build bot will figure it out by itself. So please just leave it as it is and we can get it merged - I have no access to a mac at the moment but I should be able to merge it tonight european time unless @SMillerDev gets there before me =)

**Comment by alcaeus** [comment]
@javian sounds good, thanks for the explanation! 👍

**Comment by SMillerDev** [comment]
Very subtle @javian 😉 
Thanks @alcaeus !

**Comment by javian** [comment]
@SMillerDev I need to put my manager skills to good use =D

**Pull request closed without merging**